test/blob.js

```diff
@@ -71,7 +72,7 @@ describe('hashing Blob() objects', function() {
       type: 'application/octet-stream',
       lastModified: 100000
     }));
-    var hash2 = hash(new File(new Uint8Array([1,2,3]), 'bar', {
+    var hash2 = hash(new File(new Uint8Array([1,2,3]), 'foo', {
```

Comment:
This test was faulty, as we do check for the file name, and these were not the same in the comparison.